### PR TITLE
CLAAT export tool

### PR DESCRIPTION
**Pull request by domesticmouse**
This is a tool for exporting images associated with a CLAAT Google codelab doc.

## Pre-launch Checklist

- [x] I read the [Effective Dart: Style] _recently_, and have followed its advice.
- [x] I signed the [CLA].
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-devrel channel on [Discord].

[Effective Dart: Style]: https://dart.dev/guides/language/effective-dart/style
[CLA]: https://cla.developers.google.com/
[Discord]: https://github.com/flutter/flutter/wiki/Chat

### CODE REVIEW

**Comment by domesticmouse** [comment]
PTAL @RedBrogdon 

This is ready to land. It's a useful tool for pulling the images of a codelab, which can then be resized as required.

**Review by RedBrogdon — COMMENTED** [review]
Left some nits about the readme and a few other things.

Does this need a `LICENSE` file or file headers?

File: tooling/claat_export_images/test/data/README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+## Test data
+
+The files under `exports` are created as follows:
+
+```console
+$ cd exports
+$ dart run ../../../bin/claat_export.dart -s ../../../client_secret.apps.googleusercontent.com.json -d 1MfCrv1w6aK7SLq9gmjJwXCJFQ-AX9NWJit2gq-v2Xx8
```
[the remark this answers is not in the document]
Comment (domesticmouse) [reply tooling/claat_export_images/test/data/README.md:7]: The tests do no network access. The files in `data/exports` are cached versions of the API result that this code runs on.

File: tooling/claat_export_images/README.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# CLAAT Image exporter, Dart version
+
+A tool to export the images for a CLAAT codelab.
```
[the remark this answers is not in the document]
Comment (domesticmouse) [reply tooling/claat_export_images/README.md]: Added

File: tooling/claat_export_images/README.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+  [setting up your environment]: https://developers.google.com/docs/api/quickstart/python#set_up_your_environment
+ 
+
+## Run
+
+After following the quickstart setup instructions, run the code:
+
+```shell
+dart bin/claat_export_images.dart -s client_secret.json -d 1389diNFkkLUQUVIpJ1B2XGdk8wfsPNJOGeVYZWlEhpk
```
[the remark this answers is not in the document]
Comment (domesticmouse) [reply tooling/claat_export_images/README.md]: Done

**Review by RedBrogdon — APPROVED** [review]

File: tooling/claat_export_images/test/data/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (RedBrogdon) [reply tooling/claat_export_images/test/data/README.md]: Got it. I missed that detail.

**Pull request merged**